assembl/static2/js/app/utils/tree.js

```diff
@@ -27,11 +32,91 @@ export function getChildren<T: NodeType>(rootNode: T, nodes: Array<T>): Array<T>
  */
 export function getPartialTree<T: NodeType>(nodes: Array<T>): TreeType<T> {
   let ids = nodes.map(node => node.id);
-  const roots = nodes.filter(node => node.ancestors.every(a => !ids.includes(a)));
+  const roots = nodes.filter(node => !node.ancestors || node.ancestors.every(a => !ids.includes(a)));
```

Comment:
I don't understand how a tree can have multiple roots (same for `getPartialTreeByParent` below). Maybe a comment would be welcome here to clarify this

Reply:
not a tree but a Partial Tree ;)